inst/examples/Calendar.R

```diff
@@ -1,23 +1,10 @@
 library(shiny.fluent)
 
 if (interactive()) {
-  dayPickerStrings <- jsonlite::fromJSON(
```

Comment:
why don't we need this anymore?

Reply:
We never needed this - if the `dayPickerStrings` prop is not provided, default settings are used.